examples/nodejs-nestjs/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "nest-js",
+  "version": "0.0.1",
+  "exports": "./index.js",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
+  "type": "module",
+  "scripts": {
+    "build": "nest build",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "start": "nest start",
+    "start:dev": "nest start --watch",
+    "start:debug": "nest start --debug --watch",
+    "start:prod": "node dist/main",
+    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:cov": "jest --coverage",
+    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+    "test:e2e": "jest --config ./test/jest-e2e.json"
+  },
+  "dependencies": {
+    "@arcjet/node": "^1.0.0-alpha.12",
```

Comment:
I think we'll want to add this to CI so this should be a file reference so it always pulls the latest SDK from the repo. Example at https://github.com/arcjet/arcjet-js/blob/330b3178a2ac273134d5a18d60078d651bd72f67/examples/nextjs-14-decorate/package.json#L13